src/main/java/net/sf/jabref/logic/error/MessagePriority.java

```diff
@@ -0,0 +1,28 @@
+/*  Copyright (C) 2016 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.logic.error;
+
+/**
+ * This enum will be define for three different type of message
+ * <ul>
+ * <li>log
+ * <li>output
+ * <li>exception
+ * </ul>
+ */
+public enum MessagePriority {
+    LOW, MEDIUM, HIGH;
```

Comment:
As I said before, please rename
LOW -> LOG
MEDIUM -> OUTPUT
HIGH -> EXCEPTION

and MessagePriority -> MessageType as well as LogMessageWithPriority -> LogMessage